source/blog/2022-10-19-announcing-solidus-release-policy.html.erb.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: Announcing Solidus Release Policy
+date: 2022-10-19
+tags: Solidus
+author: Solidus Core Team
+cover_image: /blog/2022/10/19/announcing-solidus-release-policy/release-policy.jpg
+---
+Community is Solidus' most important asset. As you already know, we put a lot
+of effort into keeping a high developer experience during upgrades for new
+releases. We know it's always difficult for teams to prioritize them, and
+programmers often need to carve out time from nowhere. Because of that,
+backward compatibility is one of our topmost priorities: we always try our best
+not to break anything.
+
+Nonetheless, we reckon we haven't been consistent on the timing for those
+releases. That had made some folks wonder when that loved new feature or when
+the latest Rails version would be officially integrated. Definitely, that's
+something that doesn't help to plan upgrades.
+
+We want to improve there. Because of that, we discussed and agreed on a release
+policy that we hope will help the community prepare for Solidus upgrades. As
+always in Open Source, resources are limited, but we're confident that this
+self-inflicted pressure will force us to keep pace with your expectations.
+
+Please, read the [whole document on our
+site](https://solidus.io/release_policy/), but if you're tight on time, here
+are the principal takeaways:
+
+- A quarterly new minor release.
+- Major releases should not be more than two years apart.
+- Patch releases at every backported fix.
+
+Share your ideas. Please, don't think twice about reaching us on our [Slack
```

Comment:
not sure but worth checking, `reaching us` → `reaching out to us`

Reply:
I think you're right. Done!